src/theme/palette.ts

```diff
@@ -15,6 +15,20 @@ const GREY = {
 	900: '#161C24',
 };
 
+// SETUP COLORS
+const ORANGE = {
```

Comment:
using Object.freeze make this object immutable in runtime

Reply:
If so, should other constant in palette.ts should be freeze as well ?

Reply:
just use when you need